### ISSUE TITLE
[Bug]:  inaccessible Elements in windows for Screenreader

### ISSUE THREAD

**Issue by JeanStiletto** [nextcloud/desktop#5617]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

In nextclud desktop client under windows 10 and 11 using screenreaders there are many accessibility issues. 
Tested screenreader is NVDA but think it is the same problem with Jaws or other Programs. 
 Menu structure isn't reachable with screenreaders or keyboard navigation. 
There are unlabeled buttons without accessibility tags. 
Many elements of the screen aren't visible with screenreaders. 

### Steps to reproduce

Using alt for accessing the menus doesn't works. Tabbing on elements only reads unlabeled button. Reaching konto functionality isn't possible.

### Expected behavior

When clicking on the share dialog, share by e-mail should be an option.
...

### Which files are affected by this bug

nextcloud desktop  client

### Operating system

Windows

### Which version of the operating system you are running.

11

### Package

Official SNAP package

### Nextcloud Server version

23.0.1

### Nextcloud Desktop Client version

3.5.0

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by citygabor** [comment]
Hello NextCloud Team,

I am still experiencing the same issue with the latest server and desktop versions(server:29.0.0, Desktop:3.13.2)
Once I am opening the nextcloud desktop  app from my tray icon after that I am not able to navigate in the desktop appp to login or to modify the settings.
I could only login with a sighted person help.

I have tested with JAWS, NVDA, Narrator on Windows 11. Everything is on the latest version.

Can you please give us any update on this issue?

Thanks,
Regards,
Gabor

**Comment by Rello** [comment]
Hello, 

please retest with a recent v33 client if the error still exists
